code/modules/medical/genetics/geneticsMachines.dm

```diff
@@ -18,24 +18,28 @@
 	object_flags = CAN_REPROGRAM_ACCESS
 	can_reconnect = 1
 	var/obj/machinery/genetics_scanner/scanner = null //Linked scanner. For scanning.
-	var/list/equipment = list(0,0,0,0)
-	// Injector, Analyser, Emitter, Reclaimer
+	var/list/equipment = list(
+		GENETICS_INJECTORS = 0,
+		GENETICS_ANALYZER = 0,
+		GENETICS_EMITTERS = 0,
+		GENETICS_RECLAIMER = 0,
+	)
```

Comment:
FALSE?

Reply:
I believe these are "timestamp when next available" but I'd have to double check.